packages/extension_google_sign_in_as_googleapis_auth/pubspec.yaml

```diff
@@ -6,22 +6,21 @@
 
 name: extension_google_sign_in_as_googleapis_auth
 description: A bridge package between google_sign_in and googleapis_auth, to create Authenticated Clients from google_sign_in user credentials.
-version: 2.0.0
-homepage: https://github.com/flutter/plugins/google_sign_in/extension_google_sign_in_as_googleapis_auth
+version: 3.0.0
+repository: https://github.com/flutter/packages/tree/master/packages/extension_google_sign_in_as_googleapis_auth
 
 dependencies:
   flutter:
     sdk: flutter
   google_sign_in: ^5.0.0
-  googleapis_auth: ^1.0.0
-  meta: ^1.3.0
+  googleapis_auth: ^1.1.0
```

Comment:
Do we actually needed anything from the latest release, or can we leave 1.0.0 to give clients more flexibility?

Reply:
This is only needed for the new import to be true.

I've also bumped the major version of this package to 3.0.0 so people have to opt-in to get the latest changes.

Reply:
What changes are they opting into though? If it's just this, the resolver will handle cases where someone is pinning an old googleapis_auth for some reason, I just thought we didn't need to narrow the range (I missed that the new import was added in 1.1.0, vs it already having been there before the declaration was added).

Reply:
Nah, I sometimes use major version changes for people to check the changelog (so they see the new location of the package), but I guess this is 2.0.1. 2.0.0 tests still pass, so this change should be minor from our standpoint, I think.

Reply:
> so they see the new location of the package

I would expect that ~0 people who aren't us will care where the source lives :)

Reply:
Haha fair enough, let me get this back to 2.0.1 and remove the prerelease bit from the sdk version :P